src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -773,7 +774,7 @@ public void saveTransitLines(File file) throws IOException {
                 LOG.error("Unsupported HVT type detected: {} for {} {}", route.getType(), route.getShortName(), route.getAgency().getName());
```

Comment:
Tutaj też jest użyte `route.getShortName()`, to chyba też trzeba zmienić w przypadku nulla?

Reply:
Słusznie - zmienione